docs/content-services/components/search-chip-input.component.md

```diff
@@ -0,0 +1,49 @@
+---
+Title: Search Chip Input component
+Added: v6.1.0
+Status: Active
+Last reviewed: 2023-06-01
+---
+
+# [Search Chip Input component](../../../lib/content-services/src/lib/search/components/search-chip-input/search-chip-input.component.ts "Defined in search-chip-input.component.ts")
+
+Represents a input with stacked list of chips as phrases added through input.
```

Comment:
small comment but should be rather 'an' before input word

Reply:
Fixed